MobileBuy/sample/src/main/java/com/shopify/sample/application/SampleApplication.java

```diff
@@ -158,7 +177,11 @@ public void createCheckout(final Product product, final Callback<Checkout> callb
         address.setCountryCode("US");
         checkout.setShippingAddress(address);
 
-        checkout.setEmail("something@somehost.com");
+        if (customer != null) {
```

Comment:
You should to set the address based on the Customer information as well if it is available.
Customer should have a default address in it.